thewalrus/tests/test_quantum.py

```diff
@@ -1521,3 +1522,47 @@ def test_big_displaced_squeezed_matelem():
     cutoff = 100
     probs_displaced_squeezed = probabilities(mu, cov, cutoff, hbar=1.0)
     assert np.allclose(np.sum(probs_displaced_squeezed), 1.0)
+
+@pytest.mark.parametrize("s", [0.5, 0.7, 0.9, 1.1])
+@pytest.mark.parametrize("k", [4, 6, 10, 12])
+def test_total_photon_number_distribution_values(s, k):
+    """Test that the total photon number distribution is correct when there are no losses"""
+    cutoff = 300
+    eta = 1.0
+    expected_probs = _squeezed_state_distribution(s, cutoff, N=k)
+    probs = np.array([total_photon_number_distribution(i, k, s, eta) for i in range(cutoff)])
+    assert np.allclose(expected_probs, probs)
+
+@pytest.mark.parametrize("s", [0.5, 0.7, 0.9, 1.1])
+@pytest.mark.parametrize("k", [4, 6, 10, 12])
+@pytest.mark.parametrize("eta", [0, 0.1, 0.5, 1])
+def test_total_photon_number_distribution_moments(s, k, eta):
+    """Test that the total photon number distribution has the correct mean and variance"""
+    expectation_n = characteristic_function(s = s, k = k, eta = eta, poly_corr=1, mu = 0)
+    expectation_n2 = characteristic_function(s = s, k = k, eta = eta, poly_corr=2, mu = 0)
+    var_n =expectation_n2 - expectation_n ** 2
+    expected_n = eta * k * np.sinh(s) ** 2
+    expected_var = expected_n * (1 + eta * (1 + 2 * np.sinh(s) ** 2))
+    assert np.allclose(expectation_n, expected_n)
+    assert np.allclose(expected_var, var_n)
+
+@pytest.mark.parametrize("s", [0.5, 0.7, 0.9, 1.1])
+@pytest.mark.parametrize("k", [4, 6, 10, 12])
+@pytest.mark.parametrize("eta", [0, 0.1, 0.5, 1])
+def test_characteristic_function_is_normalized(s, k, eta):
+    r"""Check that when evaluated at \mu=1 the characteristic function gives 1"""

```

Comment:
```suggestion
    r"""Check that when evaluated at mu=1 the characteristic function gives 1"""

```

Reply:
It seems to me like this function evaluates at mu=0 rather than mu=1 as is stated here. Typo? 🤔

Reply:
Good catch!